packages/components/src/Card/CardClickable.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactNode, createRef } from "react";
+
+interface ClickableCardProps {
+  onClick(event: React.MouseEvent<HTMLElement>): void;
+  className: string;
+  readonly children: ReactNode | ReactNode[];
+}
+
+const ENTER_KEY = "Enter";
+const SPACEBAR_KEY = " ";
+
+/**
+ * This is only intended to be used in the Card component.
+ * Please use `<Card onClick={onClick} />` component instead.
+ */
+export function CardClickable({
+  className,
+  onClick,
+  children,
+}: ClickableCardProps) {
+  const cardRef = createRef<HTMLDivElement>();
+
+  return (
+    <div
+      ref={cardRef}
+      data-testid="clickable-card"
+      className={className}
+      onClick={onClick}
+      onKeyUp={handleKeyup}
```

Comment:
Could we combine key-up and key-down handlers into one?

Reply:
Not with the context of firing events through space and enter since the `button`, which i'm mimicking here, fires on keyup

Reply:
That makes sense